src/io/util.cpp

```diff
@@ -0,0 +1,37 @@
+#include "aikido/io/util.hpp"
+
+namespace aikido {
+namespace io {
+
+dart::dynamics::SkeletonPtr loadSkeletonFromURDF(
+    const dart::common::ResourceRetrieverPtr resourceRetriever,
+    const std::string& uri,
+    const Eigen::Isometry3d& transform = Eigen::Isometry3d::Identity())
```

Comment:
I would prefer making the default param `Eigen::Isometry3d::identity()` be specified at the header file to make it more explicit.